app/services/foreman_ansible/playbooks_importer.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+module ForemanAnsible
+  # Imports playbooks from smart proxy
+  class PlaybooksImporter
+    include ::ForemanAnsible::ProxyAPI
+    delegate :playbooks, :playbooks_names, :to => :proxy_api
+
+    def initialize(proxy = nil)
+      @ansible_proxy = proxy
+    end
+
+    def import_playbooks(playbooks_names)
+      playbooks = playbooks(playbooks_names)
+      result = { created: {}, updated: {}, deleted: {} }
+      playbooks.each do |playbook|
+        parsed_playbook = parse_playbook playbook
+        job_template_id = JobTemplate.where(name: parsed_playbook[:name]).pick(:id)
+        if job_template_id.present?
+          updated = update_job_template(job_template_id, parsed_playbook)
+          result[:updated].merge!(updated) unless updated.nil?
+        else
+          result[:created].merge!(create_job_template(parsed_playbook))
+        end
+      end
+      result[:deleted] = delete_job_templates(playbooks_names)
+      result
+    end
+
+    def parse_playbook(playbook)
+      content = playbook['playbooks_content'].join('')
+      {
+        name: playbook['name'],
+        playbook_content: add_metadata(playbook['name']) + content,
+        vars: get_vars(content)
+      }
+    end
+
+    def add_metadata(playbook_name)
+      [
+        '<%#',
+        "  name: #{playbook_name}",
+        '  snippet: false',
+        '  job_category: Ansible Playbook - Imported',
+        '  provider_type: Ansible',
+        '  kind: job_template',
+        '  model: JobTemplate',
+        "%>\n"
+      ].join('')
```

Comment:
Shouldn't we join with `"\n"`? Optionally, we could use [HEREDOC syntax](http://graysoftinc.com/ruby-voodoo/working-with-multiline-strings)